pkg/api/aim/dashboards.go

```diff
@@ -22,10 +23,11 @@ func GetDashboards(c *fiber.Ctx) error {
 
 	var dashboards []database.Dashboard
 	if err := database.DB.
-		Preload("App").
-		Joins("LEFT JOIN apps ON apps.id = dashboards.app_id").
+		Joins(
+			"INNER JOIN apps ON dashboards.app_id = apps.id AND apps.namespace_id = ?",
+			ns.ID,
+		).
```

Comment:
the app type is now missing because it's not properly joined

Reply:
fixed.